### DISCUSSION TITLE
Ordered list 'start' number?

### DISCUSSION THREAD

**Discussion by johnsturgeon** [squidfunk/mkdocs-material#7327] [Q&A] [answered]
Is there a way to define the starting number of an ordered list?  In Jekyll I used `{:start="5"}`. I can't find anything in the docs

**Comment by facelessuser** [comment]
You may find this extension helpful which is shipped with Python Markdown: https://python-markdown.github.io/extensions/sane_lists/

**Reply by joapuiib** [reply, marked as answer]
You can enable the extension in `mydocs.yml`:
```yml
markdown_extensions:
  - sane_lists
```

**Reply by johnsturgeon** [reply]
LOL, thank you -- I had no idea it was that easy 😊

**Reply by joapuiib** [reply]
I did not know it wasn't possible by default until I saw this discussion!

I think it would be nice to have this configuration listed in the [lists documentation](https://squidfunk.github.io/mkdocs-material/reference/lists/)
